dataverse-webapp/src/main/webapp/viewMetadata.xhtml

```diff
@@ -21,9 +21,13 @@
                 </a>
                 <div id="panelCollapse#{block.index}" class="collapse #{block.first?'in':''}">
                     <script>
-                        $('#panelCollapse#{block.index}').on('shown.bs.collapse', function() {
-                            Drodb.Geo.MetadataView.initializeAll('map_#{metadataBlockVal.key.name}');
-                        });
+                        if ($('#panelCollapse#{block.index}').hasClass('in')) {
+                            DvJS.Geo.MetadataView.initializeAll('map_#{metadataBlockVal.key.name}');
```

Comment:
One more thing. We are expecting that `initializeAll` will be executed after `prepare`. So probably this script section should be at the bottom of `<div id="panelCollapse#{block.index}"`